tests/test_https_allowunmocked.js

```diff
@@ -1,53 +1,67 @@
 'use strict'
 
 const { test } = require('tap')
-const mikealRequest = require('request')
+const fs = require('fs')
+const https = require('https')
 const nock = require('../')
+const got = require('got')
 
 nock.enableNetConnect()
 
-// Do not copy tests that rely on the process.env.AIRPLANE, we are deprecating that via #1231
-test('allowUnmocked for https', { skip: process.env.AIRPLANE }, function(t) {
-  nock('https://www.google.com/', { allowUnmocked: true })
-    .get('/pathneverhit')
-    .reply(200, { foo: 'bar' })
+process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = 0
```

Comment:
This i so confusing. While trying to change, the test appears to be working without the tls boolean. Could have been fixed I set the port explicitly to 3000 instead of letting `https.createServer` assign a random port. I'll push and see what Travis says as well.

Reply:
No. Stupid me. Was changing in the wrong repo and branch...